src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -1576,7 +1576,7 @@ void DacDbiInterfaceImpl::GetStaticsBases(TypeHandle thExact,
                                          PTR_BYTE *  ppGCStaticsBase,
                                          PTR_BYTE *  ppNonGCStaticsBase)
  {
-    MethodTable * pMT = thExact.GetMethodTable();
+    PTR_MethodTable pMT = thExact.GetMethodTable();
```

Comment:
```suggestion
    MethodTable * pMT = thExact.GetMethodTable();
```
This change should not be needed and it is a small de-optimization.

Before this change, we resolved `PTR_MethodTable` to `MethodTable*` once in this method. With this change, we will be resolving `PTR_MethodTable` to `MethodTable*` twice, in each `->`.